ipalib/cli.py

```diff
@@ -1346,6 +1347,9 @@ def run(api):
     error = None
     try:
         (_options, argv) = api.bootstrap_with_global_options(context='cli')
+
+        check_client_configuration()
+
```

Comment:
Here also you should handle the exception and call sys.exit(e) otherwise the cli prints the traceback.